themes/github.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+// Original: https://raw.githubusercontent.com/PrismJS/prism-themes/master/themes/prism-ghcolors.css
+
+/*:: import type { PrismTheme } from '../src/types' */
+
+var theme /*: PrismTheme */ = {
+  plain: {
+    color: "#393A34",
+    backgroundColor: "#f6f8fa",
+    fontFamily:
```

Comment:
This will need to be removed as it's not a particular concern to prism-react-renderer what font is used 👍